pyo3-polars/src/alloc.rs

```diff
@@ -67,14 +67,16 @@ impl PolarsAllocator {
         // Do not allocate in this function,
         // otherwise it will cause infinite recursion.
         self.0.get_or_init(|| {
-            Python::with_gil(|_| {
-                unsafe {
-                    (PyCapsule_Import(ALLOCATOR_CAPSULE_NAME.as_ptr() as *const c_char, 0)
-                        as *const AllocatorCapsule)
-                        .as_ref()
-                }
+            (unsafe { Py_IsInitialized() } != 0)
```

Comment:
Should we wait here until python is initialized? In which cases is this `false/true`?

Reply:
It is always `true` if we are building a python extension module.

`cargo test` does not build a python extension module but a standalone executable, so it needs special handling.

Reply:
Thanks. That context is what I needed. :)